src/screens/TransactionConfirmationScreen.js

```diff
@@ -108,7 +118,7 @@ const TransactionConfirmationScreen = () => {
     InteractionManager.runAfterInteractions(() => {
       startPollingGasPrices();
     });
-  }, [routeParams?.openAutomatically, startPollingGasPrices]);
+  }, [routeParams.openAutomatically, startPollingGasPrices]);
```

Comment:
accessing routeParams.openAutomatically could cause an error, perhaps use the pattern of accessing routeParams?.openAutomatically as a const outside and then use that const as the dependency